rosdep/python.yaml

```diff
@@ -6369,6 +6369,12 @@ python3-pyparsing:
   gentoo: [dev-python/pyparsing]
   openembedded: [python3-pyparsing@meta-python]
   ubuntu: [python3-pyparsing]
+python3-pypng:
+  arch: [python-pypng]
+  debian: [python3-png]
+  fedora: [python3-pypng]
+  gentoo: [dev-python/pypng]
+  ubuntu: [python3-png]
```

Comment:
Unfortunately, it looks like this package isn't available in Ubuntu Xenial.  Since that is still one of our supported platforms, I think we need to change this to:

```
  ubuntu:
    '*': [python3-png]
    xenial: null
```

Reply:
done